Lib/test/test_robotparser.py

```diff
@@ -96,6 +96,18 @@ class RejectAllRobotsTest(BaseRobotTest, unittest.TestCase):
     bad = ['/cyberworld/map/index.html', '/', '/tmp/']
 
 
+class InvalidFileTest(BaseRobotTest, unittest.TestCase):
```

Comment:
This tests an empty file, which doesn't appear to be invalid from my reading. I suggest renaming to "EmptyFileTest".
```suggestion
class EmptyFileTest(BaseRobotTest, unittest.TestCase):
```